caravel/data/__init__.py

```diff
@@ -70,98 +72,102 @@ def load_energy():
-                    "target"
-                ],
-                "having": "",
-                "metric": "sum__value",
-                "row_limit": "5000",
-                "slice_id": "",
-                "slice_name": "Energy Sankey",
-                "viz_type": "sankey",
-                "where": ""
-            }
-            """))
+    slc = Slice(
+        slice_name="Energy Sankey",
+        viz_type='sankey',
+        datasource_type='table',
+        datasource_id=tbl.id,
+        params=textwrap.dedent("""\
+        {
+            "collapsed_fieldsets": "",
+            "datasource_id": "3",
+            "datasource_name": "energy_usage",
+            "datasource_type": "table",
+            "flt_col_0": "source",
+            "flt_eq_0": "",
+            "flt_op_0": "in",
+            "groupby": [
+                "source",
+                "target"
+            ],
+            "having": "",
+            "metric": "sum__value",
+            "row_limit": "5000",
+            "slice_id": "",
+            "slice_name": "Energy Sankey",
+            "viz_type": "sankey",
+            "where": ""
+        }
+        """)
     )
+    merge_slice(slc)
+    misc_dash_slices.append(slc)
 
-    merge_slice(
-        Slice(
-            slice_name="Energy Force Layout",
-            viz_type='directed_force',
-            datasource_type='table',
-            datasource_id=tbl.id,
-            params=textwrap.dedent("""\
-            {
-                "charge": "-500",
-                "collapsed_fieldsets": "",
-                "datasource_id": "1",
-                "datasource_name": "energy_usage",
-                "datasource_type": "table",
-                "flt_col_0": "source",
-                "flt_eq_0": "",
-                "flt_op_0": "in",
-                "groupby": [
-                    "source",
-                    "target"
-                ],
-                "having": "",
-                "link_length": "200",
-                "metric": "sum__value",
-                "row_limit": "5000",
-                "slice_id": "229",
-                "slice_name": "Force",
-                "viz_type": "directed_force",
-                "where": ""
-            }
-            """))
+    slc = Slice(
+        slice_name="Energy Force Layout",
+        viz_type='directed_force',
+        datasource_type='table',
+        datasource_id=tbl.id,
+        params=textwrap.dedent("""\
+        {
+            "charge": "-500",
+            "collapsed_fieldsets": "",
+            "datasource_id": "1",
+            "datasource_name": "energy_usage",
+            "datasource_type": "table",
+            "flt_col_0": "source",
+            "flt_eq_0": "",
+            "flt_op_0": "in",
+            "groupby": [
+                "source",
+                "target"
+            ],
+            "having": "",
+            "link_length": "200",
+            "metric": "sum__value",
+            "row_limit": "5000",
+            "slice_id": "229",
```

Comment:
oh good catch, I just de-dented this as is, but the slice_id should not be in there...